terraform/aws/environments/staging/3-cluster/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  required_version = ">= 0.11"
```

Comment:
Should this be 0.12?

Reply:
Yeap good catch. I will add 0.12 to error out when users try to use an older Terraform version. Thanks :)

Reply:
Done